app/models/issue.rb

```diff
@@ -64,12 +64,12 @@ def lon
     position&.x
   end
 
-  def lat_external
-    external_position.y
+  def lat_external(round: false)
+    round ? external_position.y.round : external_position.y
```

Comment:
```suggestion
  def lat_external
    external_position.y
```
Da im Model keinerlei Prüfung auf das Vorhandensein eines Koordinatenwertes erfolgt kann der Aufruf der Rundung auch direkt bei der Ausgabe erfolgen. Da bringt die Erweiterung der internen API keinen Vorteil.